StringHelper.cs

```diff
@@ -49,8 +49,12 @@ public static int CountPunctuationMark(string text)
         /// </summary>
         public static int CountSentenses(string text)
         {
-            string[] sentenses = text.Split(new char[] { '.', '!', '?' }, StringSplitOptions.RemoveEmptyEntries);
+
+
+            string[] sentenses = text.Split(new char[] { '.', '?' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
I think there might be small bug in that logic. If last sentense ends with '!'
eg. I like you. Do I like you? I like you ! - it will give you 3 sentences instead of 2.